script/gen-service-worker.js

```diff
@@ -9,11 +9,11 @@ Expects home-assistant-polymer repo as submodule of HA repo.
 Creates a caching service worker based on the CURRENT content of HA repo.
 Output service worker to build/service_worker.js
 */
```

Comment:
Yeah, not having it will slow down the loading of the UI. Let's at least store English as it's the fallback language. I don't know if we should go and cache every language, we probably should setup sw-precache to cache the other language files when requested once (but that's another PR for sure!).